src/desktop/README.md

```diff
@@ -40,15 +40,24 @@ cd trinity-wallet
 Now we need to install dependencies, such as the electron installer or the React code packager. Do this by running:
 
 ```
-yarn full-setup
+npm run full-setup
```

Comment:
The full-setup script will need to be changed as it runs yarn instead of npm

Reply:
Updated the `full-setup` script for desktop